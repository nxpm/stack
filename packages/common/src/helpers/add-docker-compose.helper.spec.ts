import { Tree } from '@nrwl/devkit'
import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing'
import { addDockerCompose } from './add-docker-compose.helper'

describe('add-docker-compose helper', () => {
  let appTree: Tree

  beforeEach(() => {
    appTree = createTreeWithEmptyWorkspace()
  })

  it('should run successfully', async () => {
    addDockerCompose(appTree)
    expect(appTree.read('docker-compose.yml').toString()).toMatchSnapshot()
  })
})
