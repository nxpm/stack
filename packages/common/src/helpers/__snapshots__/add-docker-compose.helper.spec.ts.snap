// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add-docker-compose helper should run successfully 1`] = `
"version: \\"3\\"
services:
  postgres:
    image: postgres
    ports:
      - 5432:5432
    environment:
      POSTGRES_DB: prisma
      POSTGRES_USER: prisma
      POSTGRES_PASSWORD: prisma
    volumes:
      - ./tmp/postgres:/var/lib/postgresql/data
"
`;
